models/articles.models.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/connection');
+const format = require('pg-format');
+
+const readArticlesById = (article_id) => {
+
+    return db.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then(({ rows }) => {
+    if (rows.length === 0){
+         Promise.reject({status: 404, msg : 'Not found'});
+    }
```

Comment:
Do we need to promise reject here, will PSQL not give an error if it doesn't find the article_id ?